### PR TITLE
cBlockInfo static initialisation

### PR DESCRIPTION
**Pull request by peterbell10**
Changes `cBlockInfo::Get` to use normal static initialisation and fixes the formatting in BlockInfo.cpp

### CODE REVIEW

File: src/BlockInfo.cpp [hunk 1]
```diff
@@ -6,934 +6,933 @@
...
+	Info[E_BLOCK_BIG_FLOWER                   ].m_IsSolid = false;
+	Info[E_BLOCK_BROWN_MUSHROOM               ].m_IsSolid = false;
+	Info[E_BLOCK_CARROTS                      ].m_IsSolid = false;
+	Info[E_BLOCK_CHORUS_PLANT                 ].m_IsSolid = false;
+	Info[E_BLOCK_CHORUS_FLOWER                ].m_IsSolid = false;
+	Info[E_BLOCK_COBWEB                       ].m_IsSolid = false;
+	Info[E_BLOCK_CROPS                        ].m_IsSolid = false;
+	Info[E_BLOCK_DANDELION                    ].m_IsSolid = false;
+	Info[E_BLOCK_DEAD_BUSH                    ].m_IsSolid = false;
+	Info[E_BLOCK_DETECTOR_RAIL                ].m_Is
```
Comment (worktycho) [inline src/BlockInfo.cpp]: I think msvc needs a std::move here.

**Comment by Seadragon91** [comment]
I can compile that with VS 2017. On appveyor VS 2012 is used, maybe it's to old.

**Comment by peterbell10** [comment]
Looks like I've got it working now but I have to say, using a more up to date compiler would have made things a lot simpler.

**Comment by madmaxoft** [comment]
AppVeyor is using VS2013, that is the lowest MSVC version that we currently support.

**Pull request merged**